rails-time-track/app/models/project.rb

```diff
@@ -0,0 +1,2 @@
+class Project < ApplicationRecord
```

Comment:
You should add the relation with other tables. For instances  `has_many :WeeklyProjectReport`

Reply:
Yeah, you are right. I have added all the has_many and has_many through necessary